src/workloads/execution/DensifyFillCombo.yml

```diff
@@ -0,0 +1,181 @@
+
+GlobalDefaults:
+  dbname: &db test
+  batchSize: &batchSize 30000
+  fieldName: &field "numeric"
+  index: &index
+    keys: {numeric: 1}
+    options: {name: "numeric"}
+  smallStepSize: &smallStep {^RandomInt: {min: 3, max: 3}}
+  nop: &Nop {Nop: true}
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *db
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: *batchSize
+    Document:
+      partitionKey: {^RandomInt: {min: 1, max: 100}}
+      toFillRandomType: {^Choose: {from: [
+        {^RandomInt: {min: -100, max: 100}},
+        {^RandomDouble: { min: 0.0, max: 500.0 }},
+        {^RandomDate: {min: "2021-01-01", max: "2023-01-01"}},
+        {^RandomString: {length: 15}}, null], weights: [1, 1, 1, 1, 6]}}
+      numeric: {^RandomInt: {min: 0, max: 2000}}
+    # $densify stages always add a $sort on the field that is being densified, so this phase adds
+    # indexes on the date/number fields so that we aren't performing an in-memory sort.
+    Indexes:
+    - *index
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+
+- Name: DensifyFillWithoutPartition
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - Duration: 30 seconds
+    Database: *db
+    Operations:
+    - OperationMetricsName: TestDensifyFillWithoutPartition
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [
+          $densify: {
+            field: *field,
+            range: {
+              bounds: "full",
+              step: *smallStep
+            }
+          },
+          $fill: {
+            output: {
+              toFillRandomType: {method: "locf"}
+            }
+          }
+        ]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: DensifyFillWithPartition
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Duration: 30 seconds
+    Database: *db
+    Operations:
+    - OperationMetricsName: TestDensifyFillWithPartition
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [
+          $densify: {
+            field: *field,
+            range: {
+              bounds: "partition",
+              step: *smallStep
+            },
+            partitionByFields: ["partitionKey"]
+          },
+          $fill: {
+            output: {
+              toFillRandomType: {method: "locf"},
+            },
+            partitionByFields: ["partitionKey"]
+          }
+        ]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+  - *Nop
+  - *Nop
+
+- Name: DensifyFillFullBoundsWithPartition
```

Comment:
Full/partition isn't necessary to include.

Reply:
done